website/docs/tutorial/getting-set-up/setting-up-databricks.md

```diff
@@ -157,48 +158,90 @@ For dbt to connect to the most optimized SQL experience in Databricks, you need
     <div style={{maxWidth: '400px'}}>
     <Lightbox src="/img/databricks_tutorial/images/query_check.png" title="Query Check" />
     </div>
-6. To get setup in dbt Cloud, you will now need to get the SQL Endpoint connection information and generate a user token. You can find your SQL endpoint connection information by going to the `Databricks UI > SQL > SQL Endpoints > Starter Endpoint > Connection details`. Save this information or keep this window open for the next section.
 
-    <Lightbox src="/img/databricks_tutorial/images/SQL_Endpoint_Details.png" title="Databrick SQL Endpoint Connection Information" />
+6. To ensure any users who might be working on your dbt project has access to your object, run this command.
 
-7. To generate a user token for your development credentials in dbt Cloud, click on `Settings` on the left side console (while still in the SQL part of the workspace). Click on `Personal Access Token` and provide a comment like `dbt Cloud development`. Save the token information somewhere because you will need it for the next part. 
-    <div style={{maxWidth: '400px'}}>
-    <Lightbox src="/img/databricks_tutorial/images/Generate_user_token.png" title="Generate User Token" />
-    </div>
+    ```sql 
+    grant all privileges on schema default to users;
+    ```    
 
 Congratulations! At this point, you have created a Databricks account, loaded training data, and successfully set up a SQL end point to query the data.  
 
 ## Connecting to dbt Cloud
 
-Now it’s time to connect dbt Cloud to Databricks in order to develop your dbt Project. 
+ There are two ways to connect dbt Cloud and Databricks. The first option is Partner Connect, which provides a streamlined setup to create your dbt Cloud account from within your new Databricks trial account. The second option is to create your dbt Cloud account separately and build the Databricks connection yourself. If you are looking to get started quickly, we recommend option 1. If you are looking to customize your setup from the very beginning and gain familiarity with the dbt Cloud setup flow, we recommend option 2.
 
-<Snippet src="tutorial-create-new-dbt-cloud-account" />
+### Option 1: Connect dbt Cloud and Databricks with partner connect
+
+1. In the Databricks workspace, on the left-side console: click on `Partner Connect`. 
+
+<div style={{maxWidth: '400px'}}>
```

Comment:
Thank you for using the same sizes for images! Can you indent these images in this section like we do in Option 2? Then they will be left-justified with the text in the step.

Reply:
Happy to! Thanks for catching that!